src/soap-module-options.type.ts

```diff
@@ -4,14 +4,31 @@ import { ModuleMetadata, Type } from '@nestjs/common';
 export { Client, IOptions } from 'soap';
 
 export type BasicAuth = {
+  type?: string
   username: string;
   password: string;
 };
 
+export type WSSecurityType = {
+  type: string
+  username: string;
+  password: string;
+  options?: string;
+};
+
+export type WSSecurityOptions = {
```

Comment:
This is not being used anywhere as it seems. It should be the type of WSSecurityType options?

Reply:
ohhh. my bad ... fixing ...

Reply:
DONE! .